k8s/aat/common/divorce/div-cos.yaml

```diff
@@ -28,54 +28,16 @@ spec:
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
       image: hmctspublic.azurecr.io/div/cos:prod-135ec6cc
```

Comment:
To do: update this once https://github.com/hmcts/div-case-orchestration-service/pull/484 is merged